recipe/meta.yaml

```diff
@@ -67,183 +69,83 @@ requirements:
-    script: install_lib.bat  # [win]
-    build:
-      run_exports:
-        # no idea, going with minor pin
-        - {{ pin_subpackage('libgdal', max_pin='x.x') }}
+  - name: libgdal-core
+
+  - name: libgdal-arrow-parquet
+    script: build_plugin.sh  # [unix]
+    script: build_plugin.bat  # [win]
     requirements:
       build:
         - cmake
-        # ranlib used at install time
         - {{ compiler('c') }}
         - {{ stdlib("c") }}
-        # libstdc++ is needed in requirements/run
         - {{ compiler('cxx') }}
         - pkg-config  # [not win]
         - make  # [unix]
         - ninja  # [win]
       host:
-        - blosc
-        - cfitsio
-        - expat
-        - freexl
-        - geos
-        - geotiff
-        - giflib  # [not win]
-        - hdf4
-        - hdf5
-        - libjpeg-turbo
-        - json-c  # [not win]
-        - kealib
-        - lerc
-        - libarchive
-        - libaec
-        - libcurl
-        - libdeflate
-        - libiconv
-        - libkml
-        - libnetcdf
-        - libpng
-        - libpq
-        - libspatialite
-        - libtiff
-        - libuuid  # [linux]
-        - libwebp-base
-        - libxml2
-        - lz4-c
-        - openjpeg
-        - openssl
-        - pcre2
-        - poppler
-        - postgresql
-        - proj
-        # qhull disabled because of https://github.com/conda-forge/qgis-feedstock/issues/284#issuecomment-1356490896
-        # - qhull
-        - sqlite
-        - tiledb
-        - xerces-c
-        - xz
-        - zlib
-        - zstd
+        - {{ pin_subpackage('libgdal-core', exact=True) }}
+        - libarrow
+        - libarrow-dataset  # [libarrow != 13]
+        - libparquet        # [libarrow != 13]
       run:
-        - cfitsio
-        - geotiff
-        - giflib  # [not win]
-        - json-c  # [not win]
-        - libpq
-        - libspatialite
-        - libuuid  # [linux]
-        - libwebp-base
-        - poppler
-        - postgresql
-        - proj
-        - tiledb
-        - xz
-        - zstd
+        - libgdal-core >={{ ".".join(version.split(".")[:2]) }}
     test:
-      files:
-        - test_data
-        - run_test.bat
-        - run_test.sh
+      commands:
+        - test -f ${PREFIX}/lib/gdalplugins/ogr_Arrow${SHLIB_EXT}        # [unix]
+        - test -f ${PREFIX}/lib/gdalplugins/ogr_Parquet${SHLIB_EXT}      # [unix]
+        - if not exist %LIBRARY_LIB%\gdalplugins\ogr_Arrow.dll exit 1    # [win]
+        - if not exist %LIBRARY_LIB%\gdalplugins\ogr_Parquet.dll exit 1  # [win]
+        - gdalinfo --format Arrow
+        - gdalinfo --format Parquet
     about:
-      summary: The Geospatial Data Abstraction Library (GDAL)
+      summary: Arrow and Parquet drivers for the Geospatial Data Abstraction Library (GDAL)
       license: MIT
       license_file: LICENSE.TXT
 
-  - name: libgdal-arrow-parquet
-    script: build_arrow_parquet.sh  # [unix]
-    script: build_arrow_parquet.bat  # [win]
+  - name: libgdal-openjpeg
```

Comment:
Should this be `libgdal-openjpeg` or `libgdal-jp2openjpeg`?

Reply:
Probably libgdal-jp2openjpeg to be consistent with the driver name